packages/ui/components/form/color-picker/colorpicker.tsx

```diff
@@ -57,6 +62,21 @@ const ColorPicker = (props: ColorPickerProps) => {
         }}
         type="text"
       />
+      <div className={classNames("px-1", resetDefaultValue == null && "invisible")}>
+        <Button
+          color={resetDefaultValue == "#292929" ? "primary" : "secondary"}
+          data-testid="preview-button"
+          target="_blank"
+          variant="icon"
+          rel="noreferrer"
+          StartIcon={RotateCcw}
+          tooltip="Reset to default"
+          onClick={() => {
+            setColor(fallBackHex(resetDefaultValue, false));
+            props.onChange(resetDefaultValue);
+          }}
+        />
+      </div>
```

Comment:
```suggestion
     {!resetDefaultValue &&  <div className={classNames("px-1"}>
        <Button
          color={resetDefaultValue == "#292929" ? "primary" : "secondary"}
          data-testid="preview-button"
          target="_blank"
          variant="icon"
          rel="noreferrer"
          StartIcon={RotateCcw}
          tooltip="Reset to default"
          onClick={() => {
            setColor(fallBackHex(resetDefaultValue, false));
            props.onChange(resetDefaultValue);
          }}
        />
      </div>} 
```

We ideally want this not to show in the dom.

Reply:
I've incorporated this change with one modification. 
It should be `{resetDefaultValue && ...` and not `{!resetDefaultValue && ...`

@Udit-takkar This has fixed the type-checks.